packages/libs/react-components/README.md

```diff
@@ -116,12 +110,10 @@ development of our other products, the color sets are also still in flux. In
 general you can expect each set of colors to have the following 4 options.
 Example:
 
-```
-  primaryAccent: '#2997FF', // Vibrant
-  primarySurface: '#C2E1FF', // Low contrast
-  primaryContrast: '#00498F', // Contrast
-  primaryHighContrast: '#002F5C', // High Contrast
-```
+      primaryAccent: '#2997FF', // Vibrant
+      primarySurface: '#C2E1FF', // Low contrast
+      primaryContrast: '#00498F', // Contrast
+      primaryHighContrast: '#002F5C', // High Contrast
 
```

Comment:
Why do we prefer spaces instead of backtics (```)

Reply:
I made a tiny plugin to make code blocks be fenced always.